<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="IZDT-211" ignored="4" name="Scenarios PageBuilder Test Cases" tests="3" failures="0" timestamp="09 Aug 2018 06:51:25 GMT" time="45.784" errors="0">
  <testcase name="@BeforeClass setUpOnce1" time="0.0" classname="com.r1v2.backend.pagebuilder.module.ScenarioSpecialPage">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.r1v2.backend.pagebuilder.module.ScenarioSpecialPage.setUpOnce1(ScenarioSpecialPage.java:39)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeClass setUpOnce1 -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.core.reports.ExtentReeport">
    <skipped/>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@BeforeClass setUpOnce1" time="0.0" classname="com.r1v2.backend.pagebuilder.module.ScenarioContentPage">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.r1v2.backend.pagebuilder.module.ScenarioContentPage.setUpOnce1(ScenarioContentPage.java:42)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeClass setUpOnce1 -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.core.reports.ExtentReeport">
    <skipped/>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="testPG_1" time="20.449" classname="com.r1v2.backend.pagebuilder.module.ScenarioCamapignPage"/>
  <testcase name="testPG_1" time="0.0" classname="com.r1v2.backend.pagebuilder.module.ScenarioSpecialPage">
    <skipped/>
  </testcase> <!-- testPG_1 -->
  <testcase name="testPG_1" time="0.0" classname="com.r1v2.backend.pagebuilder.module.ScenarioContentPage">
    <skipped/>
  </testcase> <!-- testPG_1 -->
  <testcase name="@BeforeTest setUp" time="0.0" classname="com.core.reports.ExtentReeport">
    <ignored/>
  </testcase> <!-- @BeforeTest setUp -->
  <testcase name="@BeforeTest setUp" time="0.0" classname="com.core.reports.ExtentReeport">
    <ignored/>
  </testcase> <!-- @BeforeTest setUp -->
  <testcase name="@AfterSuite tearDown" time="0.0" classname="com.core.reports.ExtentReeport">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDown -->
  <testcase name="@AfterSuite tearDown" time="0.0" classname="com.core.reports.ExtentReeport">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDown -->
</testsuite> <!-- Scenarios PageBuilder Test Cases -->
